### ISSUE TITLE
[a11y][Windows High Contrast Mode]: The small toggle has no focus outline when tabbed in Windows High Contrast Mode

### ISSUE THREAD

**Issue by pnayek** [carbon-design-system/carbon#16269]
### Package

carbon-components, carbon-components-react

### Browser

Chrome

### Operating System

Windows

### Package version

"carbon-components": "10.58.13", "carbon-components-react": "7.59.19"

### Automated testing tool and ruleset

IBM Accessibility Checker tool - Confirm Windows high contrast mode is supported when using CSS to include position, or alter non-decorative content

### Description

The small toggle has no focus outline when tabbed in Windows High Contrast Mode.

<img width="254" alt="image" src="https://github.com/carbon-design-system/carbon/assets/15056711/4ace6c76-296c-4402-bc7b-b886699dfa21">

### WCAG 2.1 Violation

1.1.1 Non-Text Content

### Reproduction/example

N/A

### Steps to reproduce

N/A

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
Hi there! I believe this was fixed in https://github.com/carbon-design-system/carbon/pull/12100 which appears to have been first shipped in v10.59.20. Let me know if not, we can always reopen.
